integration_test/gce/gce_testing.go

```diff
@@ -739,6 +739,31 @@ func runCommand(ctx context.Context, logger *log.Logger, stdin io.Reader, args [
 	return output, err
 }
 
+// getGcloudConfigDir returns the current gcloud configuration directory.
+func getGcloudConfigDir(ctx context.Context) (string, error) {
+	out, err := RunGcloud(ctx, log.New(io.Discard, "", 0), "", []string{"info", "--format=value[terminator=''](config.paths.global_config_dir)"})
+	if err != nil {
+		return "", err
+	}
+	return out.Stdout, nil
+}
+
+// SetupGcloudConfigDir sets up a new gcloud configuration directory.
+// This copies the "configurations" subdirectory of the context-specified
+// configuration directory into the new directory.
+// This only works on Linux.
+func SetupGcloudConfigDir(ctx context.Context, directory string) error {
+	currentConfigDir, err := getGcloudConfigDir(ctx)
+	if err != nil {
+		return err
+	}
+	// TODO: Replace with os.CopyFS() once available.
+	if out, err := runCommand(ctx, log.New(io.Discard, "", 0), nil, []string{"cp", "-r", filepath.Join(currentConfigDir, "."), filepath.Join(directory, ".")}, nil); err != nil {
```

Comment:
why are you adding a `/.` on the end of `currentConfigDir` and `directory`?

Seems unnecessary

Reply:
The `/.` on the source directory is to ensure that only the contents are copied (https://askubuntu.com/a/1151667). The `/.` on the target directory is to ensure that it exists (and is a directory).